### DISCUSSION TITLE
How do use 1.4.2 Debug

### DISCUSSION THREAD

**Discussion by TheNormalestPeep** [KadeArchive/Kade-Engine#661] [Q&A]
I'm trying top chart a song, but i'm struggling to do so. It says that my position is wherever/0, although there are about 40 sections, and when i press space to hear the song in debug nothing happens.

**Comment by ghost** [comment]
Use latest source and not 1.4.2

**Reply by TheNormalestPeep** [reply]
1.4.2 IS the latest.

**Reply by ghost** [reply]
The latest _release_ is 1.4.2. I mean the latest _source_.
1.4.2 is _really old_

**Reply by TheNormalestPeep** [reply]
Ah.
Even then, i am using the most recent commit.

**Reply by ghost** [reply]
That's the latest source.

**Comment by puyoxyz** [comment]
did you put the audio files in correctly? (has to be `.ogg` file, `Inst` and `Voices` in `assets/songs/`
